components/docked-composer/__examples__/base.jsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-console, react/prop-types */
+import React from 'react';
+
+import DockedComposer from '~/components/docked-composer';
+import Accordion from '~/components/accordion/__examples__/base';
+
+class Example extends React.Component {
+	static displayName = 'DockedComposerExample';
+
+	state = {
+		isOpen: true,
```

Comment:
This naming convention is a bit confusing, do you think `isExpanded` and `isOpen` would be a bit less ambiguous?